### PR TITLE
[# 113490903] Implement 'started_at' to display for age of stories

### PR DESCRIPTION
**Pull request by kquet**
[Tracker #113490903](https://www.pivotaltracker.com/story/show/113490903)

### CODE REVIEW

File: lib/app/public/js/color.js [hunk 1]
```diff
@@ -1,278 +0,0 @@
-var Color = function() {
```
Comment (soonahn) [inline lib/app/public/js/color.js]: What a poorly named class, it should obviously be named `Colour`

Comment (kquet) [inline lib/app/public/js/color.js]: This is correct, ask @kenberland about his poor life decisions.

**Comment by kquet** [comment]
@kenberland @suwyn 

The "Age" column on honesty shows when the story was first created. This PR fixes that.

File: lib/tracker/story.rb [hunk 2]
```diff
@@ -181,11 +193,11 @@ def latest_build
     end
 
     def age
-      ((Time.now - created_at) / 86400).to_i
+      ((Time.now - started_at) / 86400).to_i
```
Comment (soonahn) [inline lib/tracker/story.rb]: If we use 1.day instead of 86400, we could change the number of says much easier.

**Pull request merged**